vignettes/extending-jmpost.Rmd

```diff
@@ -241,6 +247,46 @@ as these are handled by the `parameters` slot of the model object as mentioned a
 The main reason for using this approach is that `jmpost` implements the priors in such a way
 that users can change them without having to re-compile the Stan model.
 
+
+## Custom Generated Quantities
+
+In order to avoid unnecessary processing, code that is only used for generated quantities is
+only included into the Stan model when explicitly trying to extract the quantities via the
+`LongitidunalQuantities` and `SurvivalQuantities` constructors.
+
+To facilitate this, when using `LongitidunalQuantities` or `SurvivalQuantities` 
+a dedicated model method `enableGQ()` is called on the longitudinal model. This model specific
```

Comment:
Generally the docs are clear. @gowerc 
This bit stuck out: If `SurvivalQuantities` is used does it really run `enableGQ()` on the longitudinal model (only)?